<launch>

  <arg name="rviz" default="true" />
  <param name="use_sim_time" value="true" />

  <node pkg="location" type="locationSiZu_multiThread" name="locationSiZu_multiThread"  output="screen">
    <param name="MapPCDFileName" value="$(find location)/map/tengxun.pcd"/>
    <param name="LiDARName" value="/velodyne_points"/>
  </node>

  <node pkg="tf" type="static_transform_publisher" name="link2_broadcaster" args="0.4 0 0.65 0 0 0 base_link velodyne 50" />
  <node pkg="tf" type="static_transform_publisher" name="link3_broadcaster" args="0.4 0 0.55 0 0 0 base_link imu 50" />
  <node pkg="tf" type="static_transform_publisher" name="link4_broadcaster" args="0.0 0 0.0 0 0 0 map odom 50" />

  <node name="MapServer" pkg="map_server" type="map_server" output="screen" args="$(find location)/map/tengxun.yaml"/>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find location)/rviz_cfg/ndt_matching.rviz" />
  </group>


</launch>
